data/opam_docs/2.1/03_faq.md

```diff
@@ -0,0 +1,446 @@
+---
+slug: faq
+title: FAQ
+description: |
+  Answers to frequently asked questions about Opam.
+---
+
+# opam FAQ
+
+> This FAQ is for general questions about opam and its usage. You may also be
+> interested in the more advanced [Tricks](tricks) for specific use-cases
+> or advanced users.
+
+#### 🐫  What is opam for?
+
+Easily installing, upgrading and managing your OCaml compiler(s), tools and
+libraries. It's made of a tool, the
+[opam package manager](https://github.com/ocaml/opam), and a
+community-maintained
+[package repository](https://github.com/ocaml/opam-repository).
+
+Note that the tool itself is not specific to OCaml at its core, and could be
+used for different systems using specific repositories (e.g. for the
+[Coq theorem prover](http://coq.io/opam/))
```

Comment:
```suggestion
Note that the tool itself is not specific to OCaml at its core, and it could be
used for different systems using specific repositories (e.g., for the
[Coq theorem prover](http://coq.io/opam/))
```